machine.go

```diff
@@ -880,3 +879,28 @@ func (m *Machine) waitForSocket(timeout time.Duration, exitchan chan error) erro
 		}
 	}
 }
+
+// Set up a signal handler to pass through to firecracker
+func (m *Machine) setupSignals() {
+	signals := m.Cfg.ForwardSignals
+
+	if len(signals) == 0 {
+		return
+	}
+
+	m.logger.Debugf("Setting up signal handler: %v", signals)
+	sigchan := make(chan os.Signal)
```

Comment:
Can you make `sigchan` have length greater than or equal to `len(signals)`?  From [the `signal.Notify` documentation](https://golang.org/pkg/os/signal/#Notify):

> Package signal will not block sending to c: the caller must ensure that c has sufficient buffer space to keep up with the expected signal rate. For a channel used for notification of just one signal value, a buffer of size 1 is sufficient.